<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LTSGraph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ltsgraph.qrc">
    <normaloff>:/ltsgraph/icons/ltsgraph.ico</normaloff>:/ltsgraph/icons/ltsgraph.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin" stdset="0">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="widgetLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actOpenFile"/>
    <addaction name="actExportImage"/>
    <addaction name="separator"/>
    <addaction name="actImport_XML"/>
    <addaction name="actExport_XML"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="act3D"/>
    <addaction name="separator"/>
    <addaction name="actLayoutControl"/>
    <addaction name="actVisualization"/>
    <addaction name="actInformation"/>
    <addaction name="actOutput"/>
    <addaction name="separator"/>
    <addaction name="actFullscreen"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actExplorationMode"/>
    <addaction name="actLayout"/>
    <addaction name="actReset"/>
    <addaction name="actionAdvancedSpringlayout"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockOutput">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="mcrl2::gui::qt::LogWidget" name="dockWidgetOutput"/>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actLayoutControl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Layout control</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actOpenFile">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act3D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actReset">
   <property name="text">
    <string>Reset viewpoint</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actExportImage">
   <property name="text">
    <string>&amp;Export image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actImport_XML">
   <property name="text">
    <string>Import XML</string>
   </property>
  </action>
  <action name="actExport_XML">
   <property name="text">
    <string>Export XML</string>
   </property>
  </action>
  <action name="actVisualization">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visualization</string>
   </property>
  </action>
  <action name="actInformation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mCRL2 Output</string>
   </property>
  </action>
  <action name="actFullscreen">
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
  </action>
  <action name="actExplorationMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Exploration mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAdvancedSpringlayout">
   <property name="text">
    <string>Advanced Springlayout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mcrl2::gui::qt::LogWidget</class>
   <extends>QWidget</extends>
   <header>mcrl2/gui/logwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ltsgraph.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onResetClicked()</slot>
  <slot>onWidgetResized(float,float)</slot>
  <slot>onExit()</slot>
 </slots>
</ui>
